R/applications.R

```diff
@@ -133,7 +133,7 @@ showLogs <- function(appPath = getwd(), appFile = NULL, appName = NULL,
                      account = NULL, entries = 50, streaming = FALSE) {
 
   # determine the log target and target account info
-  target <- deploymentTarget(appPath, appName, account)
+  target <- deploymentTarget(appPath, appName, NULL, account)
```

Comment:
Are you purposely passing `appName` to the `appTitle` parameter of `deploymentTarget` (`deploymentTarget <- function(appPath, appTitle, appName, account, server = NULL)`)?